packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -108,10 +109,16 @@ export class HbarSpendingPlanConfigService {
    * @private
    */
   private loadSpendingPlansConfig(): SpendingPlanConfig[] {
-    const configPath = findConfig(this.SPENDING_PLANS_CONFIG_FILE);
+    const configPath = findConfig(this.SPENDING_PLANS_CONFIG);
```

Comment:
Can clean this up. 
If the config is undefined then we throw undefined error.
Beyond that it's either a file or valid json.
If the file path exist try and parse.
If not then try for a valid JSON parse.
If both cases fail then the config value is malformed.
Right now the validation paths are muddled up

Reply:
Done.